DigitalLearningSolutions.Data.Tests/TestHelpers/ProgressTestHelper.cs

```diff
@@ -69,5 +69,15 @@ FROM Progress
                 new { progressId }
             ).Single();
         }
+
+        public string GetAdminFieldAnswersByProgressId(int progressId)

```

Comment:
This name isn't really accurate to what it does, we're only getting the first answer.

Reply:
changed to GetAdminFieldAnswer1ByProgressId